src/lambdas/automation-report/src/generateReport.ts

```diff
@@ -117,7 +121,7 @@ const generateCsv = (date: Date, forces: Forces): string => {
 
   addLine("national", "National Average")
 
-  //return lines.join("\n")
+  // return lines.join("\n")
```

Comment:
do we still need this to be commented out?

Reply:
heh no - lint:fix sorted it